### PR TITLE
Implement home screen

### PR DESCRIPTION
**Pull request by KozlovaAngelika**
Implemented home screen

### CODE REVIEW

File: src/screens/Home/index.tsx [hunk 1]
```diff
@@ -1,19 +1,100 @@
...
 const Home: React.FunctionComponent = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [dataForDisplay, setDataForDisplay] = useState(DATA);
+  const keyExtractor = (item: Movie): string => item.id;
+  const renderItem = ({ item }: RenderItemProps): React.ReactElement => (
+    <MovieTile title={item.title} imgSrc={item.image} />
+  );
+  const searchMovie = debounce((value: string) => {
+    setDataForDisplay(
+      DATA.filter((item: { title: string }) =>
```
Comment (brusnev-egor) [inline src/screens/Home/index.tsx:78]: move to it to const

File: src/screens/Home/index.tsx [hunk 2]
```diff
@@ -1,19 +1,100 @@
...
+  },
+];
+interface RenderItemProps {
+  item: Movie;
+}
 const Home: React.FunctionComponent = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [dataForDisplay, setDataForDisplay] = useState(DATA);
+  const keyExtractor = (item: Movie): string => item.id;
+  const renderItem = ({ item }: RenderItemProps): React.ReactElement => (
```
Comment (brusnev-egor) [inline src/screens/Home/index.tsx:73]: there should not be RenderItemProps
it should be (item: Movie) => <MovieTile .... />

File: src/screens/Home/index.tsx [hunk 3]
```diff
@@ -1,19 +1,97 @@
...
+  },
+  {
+    id: 'tt9253866',
+    title: 'Our Planet',
+    image:
+      'https://m.media-amazon.com/images/M/MV5BN2I1ZjA5YjQtYmQ0ZS00ZmE1LTk1ZjktNTQ5ODIzY2JiZDdhXkEyXkFqcGdeQXVyNjg2NjQwMDQ@._V1_UX128_CR0,3,128,176_AL_.jpg',
+    imDbRating: '9.2',
+  },
+];
+
```
Comment (i-nemets) [inline src/screens/Home/index.tsx:66]: I think it's better to transfer temporary data to a separate file

File: src/screens/Home/index.tsx [hunk 4]
```diff
@@ -1,19 +1,97 @@
...
 const Home: React.FunctionComponent = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [dataForDisplay, setDataForDisplay] = useState(DATA);
```
Comment (i-nemets) [inline src/screens/Home/index.tsx:69]: i think better write in state initial value = '' and after setDataForDisplay(DATA)

File: src/screens/Home/styles.ts [hunk 5]
```diff
@@ -1,10 +1,14 @@
-import { StyleSheet } from 'react-native';
+import { Platform, StatusBar, StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
-    paddingVertical: 20,
+    flex: 1,
+    paddingTop: Platform.OS === 'android' ? StatusBar.currentHeight : 0,
```
Comment (brusnev-egor) [inline src/screens/Home/styles.ts:6]: do you need this?

Comment (KozlovaAngelika) [reply src/screens/Home/styles.ts]: removed unnecessary styles

**Review by brusnev-egor — APPROVED** [review]

**Review by i-nemets — APPROVED** [review]

**Pull request merged**